.github/workflows/ollama.yml

```diff
@@ -33,16 +33,18 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         python-version: ["3.9","3.10","3.11"]
-    services:
-      ollama:
-        image: ollama/ollama:latest
-        options: --name ollama
-        ports:
-          - 11434:11434
         
     steps:   
       - uses: actions/checkout@v4
 
+      - name: Install Ollama and pull the required models
+        run: |
+          curl -fsSL https://ollama.com/install.sh | sh
+          ollama serve &
+          sleep 10
```

Comment:
is there a health check for serve which we can use to see if the server is up? That might be a bit more robust than the hard coded sleep. Up to you though

Reply:
good idea. Done!